Docs/Doxygen/doxygen.conf

```diff
@@ -2037,7 +2037,15 @@ SEARCH_INCLUDES        = YES
 # preprocessor.
 # This tag requires that the tag SEARCH_INCLUDES is set to YES.
 
-INCLUDE_PATH           =
+INCLUDE_PATH           = ../../Src/Amr      \
```

Comment:
Could you tell me why need to do this now?  Should it also include all the directories recursively?

Reply:
Somehow it did not find some includes (already) - we just ignored it as warnings before it seems.

Reply:
> Note that the INCLUDE_PATH is not recursive, so the setting of [RECURSIVE](https://www.doxygen.nl/manual/config.html#cfg_recursive) has no effect here.

My point is we probably need to include ../../Src/Base/Parser etc. as well, if this is needed.

```
../../Src/Base \
../../Src/Base/Parser \
../../Src/SDC \
../../Src/Amr \
../../Src/EB \
../../Src/Particle \
../../Src/Boundary \
../../Src/Extern \
../../Src/Extern/HYPRE \
../../Src/Extern/hpgmg \ \
../../Src/Extern/HDF5 \
../../Src/Extern/PETSc \
../../Src/Extern/amrdata \
../../Src/Extern/Conduit \
../../Src/Extern/SUNDIALS
../../Src/Extern/ProfParser \
../../Src/Extern/SENSEI \
../../Src/Extern/SWFFT \
../../Src/AmrCore \
../../Src/LinearSolvers \
../../Src/LinearSolvers/MLMG \
../../Src/LinearSolvers/OpenBC \
../../Src/F_Interfaces \
../../Src/F_Interfaces/Octree \
../../Src/F_Interfaces/Base \
../../Src/F_Interfaces/Particle \
../../Src/F_Interfaces/AmrCore \
../../Src/F_Interfaces/LinearSolvers
```

Reply:
I didn't add it yet because I saw no futher warnings, but why not! :)

Reply:
Thanks for the list, updated accordingly :)

We excluded F_Interfaces/ so far, so I kept it that way for now.